plugins/catalog-backend/src/ingestion/types.ts

```diff
@@ -68,3 +69,69 @@ export type ReadLocationError = {
   location: LocationSpec;
   error: Error;
 };
+
+//
+// LocationAnalyzer
+//
+
+export type LocationAnalyzer = {
```

Comment:
Should give this one a nice JsDoc comment, what's the purpose of it etc.

Using JsDoc/TsDoc (`/** */`) comments will make the docs show up in editors as well, which is 👌

Reply:
Added, although quite minimal